src/types.js

```diff
@@ -195,43 +195,146 @@ export type TaskChange = {
   external_id?: string,
 };
 
-/* Stories */
-
-export type StoryType = 'bug' | 'chore' | 'feature';
+/* Identity */
 
-export type Story = {
+export type Identity = {
   entity_type: string,
+  name: string | null,
```

Comment:
I did that because in typescript cares (a lot) about the difference between `null` and `undefined`, since typescript uses type guards and we have `typeof null === 'object'`.

However, it appears that Clubhouse's API never produces any `undefined` value. In that case, the best to do would probably to respect Flow's convention and to add on line `.replace('?: ', 'null | ')` in `scripts/ts-generate-definition.js`, at line 34.
Do you think it would work?

Reply:
It looks good for returned types, but bad for sent types like `EpicChange`. I'm hesitant.